data_ingestion/03_Ingest_data_with_Redshift_v3.ipynb

```diff
@@ -0,0 +1,1956 @@
+{
```

Comment:
<p>Typo: </p><blockquote>and below is a helpful function helping you <strong>checking</strong> where the copying process went wrong.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>